CONTRIBUTING.md

```diff
@@ -1,44 +1,185 @@
+  * [Setting up your local environment](#setting-up-your-local-environment)
+    * [Step 1: Fork](#step-1-fork)
+    * [Step 2: Branch](#step-2-branch)
+  * [The Process of Making Changes](#the-process-of-making-changes)
+    * [Step 3: Code](#step-3-code)
+    * [Step 4: Commit](#step-4-commit)
+      * [Commit message guidelines](#commit-message-guidelines)
+    * [Step 5: Rebase](#step-5-rebase)
+    * [Step 6: Test](#step-6-test)
+    * [Step 7: Push](#step-7-push)
+    * [Step 8: Opening the Pull Request](#step-8-opening-the-pull-request)
+    * [Step 9: Discuss and Upadte](#step-9-discuss-and-update)
+      * [Approval and Request Changes Workflow](#approval-and-request-changes-workflow)
+    * [Step 10: Landing](#step-10-landing)
+  * [Reviewing Pull Requests](#reviewing-pull-requests)
+* [Additional Notes](#additional-notes)
+  * [Commit Squashing](#commit-squashing)
+  * [Getting Approvals for your Pull Request](#getting-approvals-for-your-pull-request)
+  * [CI Testing](#ci-testing)
+  * [Waiting Until the Pull Request Gets Landed](#waiting-until-the-pull-request-gets-landed)
+  * [Check Out the Collaborator's Guide](#check-out-the-collaborators-guide)
+  * [Helpful Resources](#helpful-resources)
+* [Developer's Certificate of Origin 1.1](#developers-certificate-of-origin-11)
+
 ## Code of Conduct
 
-Please read the
-[Code of Conduct](https://github.com/nodejs/TSC/blob/master/CODE_OF_CONDUCT.md)
-which explains the minimum behavior expectations for Node.js contributors.
+The Node.js project has a [Code of Conduct][] that *all* contributors are
+expected to follow. This code describes the *minimum* behavior expectations
+for all contributors.
 
-## Issue Contributions
+As a contributor to Node.js, how you choose to act and interact towards your
+fellow contributors, as well as to the community, will reflect back not only
+on yourself but on the project as a whole. The Code of Conduct is designed and
+intended, above all else, to help establish a culture within the project that
+allows anyone and everyone who wants to continue to feel safe doing so.
 
-When opening issues or commenting on existing issues, please make sure
-discussions are related to concrete technical issues with Node.js.
+Should any individual act in any way that is considered in violation of the
+[Code of Conduct][], corrective actions will be taken. It is possible, however,
+for any individual to *act* in such a manner that is not in violation of the
+strict letter of the Code of Conduct guidelines while still going completely
+against the spirit of what that Code is intended to accomplish.
 
-* For general help using Node.js, please file an issue at the
-[Node.js help repository](https://github.com/nodejs/help/issues).
+Open, diverse and inclusive communities live and die on the basis of trust.
+Contributors can disagree with one another so long as they trust that those
+disagreements are in good faith and everyone is working towards a common goal.
```

Comment:
I would add an `else` clause, something like:
```markdown
If you do not feel comfortable with these statements, or feel that you will not be able to abide by our CoC,
we respectfully request that you refrain from active participating in the Node.js project
There are many ways where you could still help the Node.js ecosystem, for example create a package,
write a blog, or join a satellite project.
```

Reply:
I see what you're going for but I think there may be a better approach... lemme stew on it.